src/pages/docs/guides/pubsub/push-notifications.mdx

```diff
@@ -0,0 +1,526 @@
+const platformToken = await getPlatformPushToken(); // Platform-specific
+await fetch('https://example.com/register-device', {
+  method: 'POST',
+  body: JSON.stringify({ 
+    userId: currentUser.id,
+    platformToken,
+    platform: 'ios' // or 'android', 'web'
+  })
+});
+
+// On your server: Register with Ably
+app.post('/register-device', async (req, res) => {
+  const { userId, platformToken, platform } = req.body;
+  
+  // Register device with Ably
+  await ablyClient.push.admin.deviceRegistrations.save({
+    id: generateDeviceId(userId, platform),
+    platform: platform,
+    formFactor: 'phone',
+    clientId: userId,
+    push: {
+      recipient: {
+        transportType: platform === 'ios' ? 'apns' : 'fcm',
+        deviceToken: platformToken
+      }
+    }
+  });
+  
+  res.json({ success: true });
+});
+```
+</Code>
+
+Server-side activation is preferred when:
+
+* You need strict control over who can register devices.
+* You want to validate user identity before enabling notifications.
+* You're migrating from another push provider and already have server-side device management.
+* You want to use FCM for both Android and iOS devices for unified push messaging.
+
+## Authentication and security
+
+Push notifications require careful permission management. Ably uses [capability-based authentication](/docs/auth/capabilities) to control what clients can do.
+
+### Capabilities
+
+Push notifications use two main capabilities:
+
+* **push-subscribe:** Allows a device to manage its own registration and subscribe to channels. Suitable for end user devices.
+* **push-admin:** Full access to push admin API - register any device, subscribe any device to channels, send direct notifications. Only for servers.
+
+
+Example capabilities:
+<Code>
+```javascript
+// Subscribe to all channels, and push-subscribe for "news:" channels; suitable for a client:
+{
+  '*': ['subscribe'],                        // Can subscribe to realtime messages
+  'news:*': ['subscribe', 'push-subscribe']  // Can also subscribe to push notifications on news channels
+}
+
+// For your server with push-admin
+{
+  '*': ['subscribe', 'publish', 'push-admin']  // Full push admin access
+}
+```
+</Code>
+
+Capabilities can be set for API keys, Ably tokens, and JWT tokens. Read more about [authentication with Ably](/docs/auth).
+
+If you are using channel-based push notifications, be aware that everyone with `publish` capability can publish messages to that channel, including sending push notifications. We recommend that only your server has `publish` capability on channels you want to use for push notifications.
+
+### Security considerations
+
+Follow these practices:
+
+* Never give clients `push-admin` capability. They could manage registrations and subscriptions for push notifications for any device.
+* Do not give `publish` capability to clients on channels used for push notifications. This could allow any clients to publish push notifications to that channel.
```

Comment:
Is this a hard no, or just a recommendation? Is there a case where someone might _want_ this?

Reply:
My take is that I can't see a use case to be honest, it's all risky to allow it. So hard no, unless you know what you're doing, in which case you'll ignore this sentence anyway (and it includes why not so I'd leave it in). More trouble than worth?